pkg/apis/application/v1alpha1/types.go

```diff
@@ -62,6 +62,8 @@ type OperationState struct {
 	SyncResult *SyncOperationResult `json:"syncResult,omitempty" protobuf:"bytes,4,opt,name=syncResult"`
 	// RollbackResult is the result of a Rollback operation
 	RollbackResult *SyncOperationResult `json:"rollbackResult,omitempty" protobuf:"bytes,5,opt,name=rollbackResult"`
+	// LastUpdateTime contains the timestamp of last operation status update
+	LastUpdateTime metav1.Time `json:"lastUpdateTime" protobuf:"bytes,6,opt,name=lastUpdateTime"`
```

Comment:
Is this a startTime, endTime or mTime? If mTime I think it might be clearer to name it such.